src/vec.rs

```diff
@@ -48,14 +48,14 @@ impl<T: MetricVecBuilder> MetricVecCore<T> {
         let mut m = MetricFamily::default();
         m.set_name(self.desc.fq_name.clone());
         m.set_help(self.desc.help.clone());
-        m.set_field_type(self.metric_type);
+        m.set_field_type_(self.metric_type);
```

Comment:
Why is there a `_` suffix?

Reply:
For enums, the `set_field_type` method is already generated by Prost, so we need to use a different name for the version we generate.